yt/frontends/ramses/tests/test_outputs.py

```diff
@@ -221,9 +222,9 @@ def test_ramses_sink():
         "particle_prop_0_1",
         "particle_prop_0_2",
         "particle_prop_0_3",
-        "particle_prop_1_0",
-        "particle_prop_1_1",
-        "particle_prop_1_2",
+        "particle_prop_0_4",
```

Comment:
This seems like a bigger change?

Reply:
Yes ; the previous behavior was inconsistent with RAMSES internals:
```fortran
     ! Write sink_stat
     do idim=1,ndim*2+1
        do ilevel=levelmin,nlevelmax
           do i=1,nsink
              xdp(i)=sink_stat(i,ilevel,idim)
           end do
           write(ilun) xdp
        enddo
     enddo
```
so the `sink_stats` which are named "`particle_prop_ilvl_idim`" in yt should have `idim=0..2*ndim` and `ilvl=0..levelmax` (assuming arrays are indexed from 0 instead of 1).

If you're unhappy with this bugfix being included in this PR, I can open another (bugfix) one.

Reply:
Nope, not unhappy.